App/Logic/AnalyticsLogic.swift

```diff
@@ -199,11 +300,76 @@ extension Logic.Analytics {
   }
 
   /// Updates the opportunity window for the event without exposure
-  struct UpdateEventWithoutExposureOppurtunityWindow: AppStateUpdater {
+  struct SetEventWithoutExposureOpportunityWindow: AppStateUpdater {
     let window: AnalyticsState.OpportunityWindow
 
     func updateState(_ state: inout AppState) {
       state.analytics.eventWithoutExposureWindow = self.window
     }
   }
+
+  /// Sets the opportunity window for the dummy analytics traffic
+  struct SetDummyTrafficOpportunityWindow: AppStateUpdater {
+    let dummyTrafficStochasticDelay: Double
+    let now: Date
+
+    func updateState(_ state: inout AppState) {
+      let windowStart = self.now.addingTimeInterval(self.dummyTrafficStochasticDelay)
+      let windowDuration = AnalyticsState.OpportunityWindow.secondsInDay
+      state.analytics.dummyTrafficOpportunityWindow = .init(windowStart: windowStart, windowDuration: windowDuration)
+    }
+  }
+}
+
+// MARK: Helpers
+
+private extension AnalyticsRequest.Body {
+  /// Creates a dummy request
+  static func dummy(deviceTokenLength: Int) -> Self {
+    return Self(
+      province: Province.allCases.randomElement() ?? AppLogger.fatalError("No provinces defined"),
+      exposureNotificationStatus: ExposureNotificationStatus.randomCase(),
+      pushNotificationStatus: UNAuthorizationStatus.randomCase(),
+      riskyExposureDetected: Bool.random(),
+      deviceToken: String.random(length: deviceTokenLength)
+    )
+  }
+}
+
+private extension ExposureNotificationStatus {
+  static let allCases: [Self] = [
+    .authorized, .authorizedAndActive, .authorizedAndBluetoothOff, .authorizedAndInactive, .notAuthorized, .restricted, .unknown
+  ]
+
+  /// Random case implementation.
+  /// This is to avoid making `ExposureNotificationStatus` conform `CaseIterable` just for this specific use case.
+  static func randomCase() -> Self {
+    Self.allCases.randomElement() ?? AppLogger.fatalError("No statuses defined")
+  }
+}
+
+private extension UNAuthorizationStatus {
+  static let allCases: [Self] = [.authorized, .denied, .notDetermined, .provisional]
+
+  /// Random case implementation.
+  /// This is to avoid making `UNAuthorizationStatus` conform `CaseIterable` just for this specific use case.
+  static func randomCase() -> Self {
+    Self.allCases.randomElement() ?? AppLogger.fatalError("No statuses defined")
+  }
+}
+
+private extension String {
```

Comment:
Remove and use `Data.random().base64...`

Reply:
Deleted for the reason stated above